src/Munchkin.Core.Cards/Doors/Monsters/Crabs.cs

```diff
@@ -10,9 +13,14 @@ public Crabs() : base("Crabs", 1, 1, 1, 0, false)
         {
         }
 
-        public override Task BadStuff(Table gameContext)
+        public override Task BadStuff(Table state)
         {
-            throw new System.NotImplementedException();
+            state.Players.Current.Equipped
+                .OfType<PermanentItemCard>()
+                .Where(x => x.WearingType == EWearingType.Armor && x.WearingType == EWearingType.Footgear)
```

Comment:
Should be || instead of &&. Permanent Item cannot be Armor and Footgear at the same time.